packages/fluentui/e2e/tests/dialogPreventScroll-test.ts

```diff
@@ -0,0 +1,58 @@
+import { selectors } from './dialogPreventScroll-example';
+
+const outerClose = `#${selectors.outerClose}`;
+const outerTrigger = `#${selectors.outerTrigger}`;
+const innerClose = `#${selectors.innerClose}`;
+const innerTrigger = `#${selectors.innerTrigger}`;
+
+const hiddenOverflow = 'overflow: hidden';
+const unsetOverflow = 'overflow: unset;';
+const dialogsCounterAttribute = 'fluent-dialogs-count';
```

Comment:
Let's avoid this as it actually tests internals: we want only to ensure that proper styles will be present.

Reply:
Done